Makefile

```diff
@@ -172,8 +172,22 @@ dev-start:
 	export DOS_DB_HOST=localhost
 	python manage.py runserver 0.0.0.0:8080
 
+dev-create-user:
+	password=$$(make secret-random)
+	cd $(APPLICATION_DIR)
+	export API_DB_HOST=localhost
+	export DOS_DB_HOST=localhost
+	python manage.py user test $$password
+
+dev-smoke-test:
+	token=$$(make dev-create-user)
+	service_id=153455
```

Comment:
A bit pedantic, but this is the service UID.

Reply:
Good point, as discussed today we are going to change our interface to use ID but let's keep that conversation in mind as we need to make sure there is no confusion in the code.